### ISSUE TITLE
First line of command overwriting prompt and looping around. 

### ISSUE THREAD

**Issue by thedonquixotic** [zeit/hyper#2930]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  Windows 10 running Ubuntu for Linux subsystem layer
- **Hyper.app version**: 2.0 Stable
- **Link of a [Gist](https://gist.github.com/thedonquixotic/859576c4972005461232e637a8d2336f)
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: 

## Issue
So basically I changed my .profile settings to have a custom CLI prompt and now it's acting weird. examples below. The first line of the command begins overlapping the prompt....Specific reason why?

![image](https://user-images.githubusercontent.com/9451726/39204413-523204c4-47bd-11e8-8135-ff3eeaafc124.png)

![image](https://user-images.githubusercontent.com/9451726/39204468-7dddb7e4-47bd-11e8-887d-dddd0725ff03.png)

![image](https://user-images.githubusercontent.com/9451726/39204488-87930c08-47bd-11e8-970c-bc9c3449b634.png)

**Comment by kayandrae07** [comment]
@albinekb Thank you for opening this. I was just about to do so. For more context, here's a video 

[![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/5YQ8ELw4VQI/0.jpg)](http://www.youtube.com/watch?v=5YQ8ELw4VQI)

**Comment by Dracunos** [comment]
I'm not sure if this is the same issue, but I'm getting issues clearing the screen as well..

On win7, just downloaded the program today. The only config I've changed was setting the terminal to powershell, but it was happening with cmd, too.

I noticed the screen not properly clearing when connecting to an ssh application using git's ssh.exe (nethack@alt.org), and noticed it again when exiting cygwin bash:

![example](https://i.imgur.com/Mxlmgwu.png)

**Comment by thedonquixotic** [comment]
Hey bumping this because I just installed all my software on a new machine and when I copied over my .bashrc file my CLI prompt was changed to my custom one and this bug appeared again. It's enough to make me think I should just forgo having a custom CLI prompt, but.... also, seems kind of like something that should be fixed? IDK. I'm not really a good developer. I'm just a designer who fucks around with code for fun.

**Comment by thedonquixotic** [comment]
Hey guys, just wanted to say that I figured this out! The issue was that vscode was adding a trailing whitespace/newline to my .profile file and that was causing the weird behavior. FIXED! WOO!

**Comment by jvzaniolo** [comment]
> Hey guys, just wanted to say that I figured this out! The issue was that vscode was adding a trailing whitespace/newline to my .profile file and that was causing the weird behavior. FIXED! WOO!

Hey man, could you be more specific ? Where can I delete this trailing whitespace/newline?

**Comment by wowpoppy** [comment]
![image](https://user-images.githubusercontent.com/66640708/109457991-25019880-7a86-11eb-81f9-80b60ae3835f.png)
The issue is still present on Hyper 3.0.2 using `WSL 2` and `tmux` on Windows 10, all latest at the moment. @jcklpe would you reopen the issue or should I create a new one?

As you may see, prompt, that should be on the right `tmux` pane, for some reason is rendering over `vim`. `vim`'s windows itself loses it's content and each line has to be re-rendered: `tabline`, code lines, `statusline`, command line. Running `clear` in the shell clears the left pane again and renders the prompt at the very beginning of the `Hyper` window.

Please tell me what info should I provide and where do I get logs and everything.

My `.hyper.js`: https://gist.github.com/wowpoppy/404b74d0b57c5490a4f3e93d735d54ca

**Comment by jcklpe** [comment]
Probably worth a new issue.
